xapian-core/tests/api_cluster.cc

```diff
@@ -29,14 +29,39 @@
 #include "testsuite.h"
 #include "testutils.h"
 
+static void
+make_stemmed_cluster_db(Xapian::WritableDatabase &db, const std::string &)
+{
+    static const char* const test_strings[] = {
+	"This line is about a cluster. Cluster is important and is everywhere",
+	"We need to search for special cluster. Cluster cluster cluster",
+	"Computer cluster is a special example of a cluster. Used to search fast",
+	"Another example of cluster is a star cluster. Star cluster has a lot of stars"
+    };
+
+    Xapian::TermGenerator indexer;
+    Xapian::Stem stemmer("english");
+    indexer.set_stemmer(stemmer);
+    unsigned int count = sizeof(test_strings)/sizeof(test_strings[0]);
```

Comment:
Missing spaces around `/`.

It'd be neater to use C++11's range-based `for` though:

```
    for (const std::string& document_data : test_strings) {
        // ...
    }
```